examples/telemetry.py

```diff
@@ -0,0 +1,24 @@
+from pteranodon import SimpleDrone
+
+
+def run():
+    drone = SimpleDrone("udp://:14540")
+
+    drone.logger.info("--- View battery")
+    drone.telemetry.battery()
```

Comment:
I think you should be printing these? 
```suggestion
    drone.logger.info(drone.telemetry.battery())
```